pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -983,13 +983,13 @@ class Build {
                                 updateGithubCommitStatus("PENDING", "Build Started")
                             }
                             if (useAdoptShellScripts) {
-                                context.println "[CHECKOUT] Checking out to AdoptOpenJDK/openjdk-build to use their bash scripts..."
-                                repoHandler.checkoutAdopt()
+                                context.println "[CHECKOUT] Checking out to AdoptOpenJDK/openjdk-build to use their shell scripts..."
+                                repoHandler.checkoutBuild()
                                 context.sh(script: "./build-farm/make-adopt-build-farm.sh")
                                 context.println "[CHECKOUT] Reverting pre-build AdoptOpenJDK/openjdk-build checkout..."
-                                context.checkout context.scm
+                                repoHandler.checkoutUser()
                             } else {
-                                context.println "[INFO] Executing user bash scripts..."
+                                context.println "[INFO] Executing user shell scripts..."
```

Comment:
Not sure the build-farm/make-adopt-build-farm.sh is going to be found as checkoutUser() checks out user pipelines repo, not a user build repo?